### PR TITLE
docs: MAINTAINERS file and LGTM configuration

### PR DESCRIPTION
**Pull request by drjova**
Signed-off-by: Harris Tzovanakis <me@drjova.com>

### CODE REVIEW

**Comment by jirikuncar** [comment]
LGTM

File: .lgtm [hunk 1]
```diff
@@ -0,0 +1,3 @@
+approvals = 1
+pattern = "(?i)LGTM"
+self_approval_off = true
```
Comment (tiborsimko) [inline .lgtm]: Note that this has to be `false` since you will be the only maintainer of the package.

Comment (drjova) [inline .lgtm]: changed to ```false```

**Comment by tiborsimko** [comment]
LGTM

**Comment by jirikuncar** [comment]
LGTM

**Pull request merged**